src/components/home/HomeProducts/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import Product from "../../Nuclie/Product";
+
+import { useData } from "../../../Context/StateContext";
+
+const HomeProducts = () => {
+  const { state } = useData();
+  const products = state.products.products;
+  return (
+    <section className='pd-container'>
+      <div className='pd-nav'>
+        <h1>Products</h1>
+        <a href='./screens/product.html' className='link-btn'>
+          See All
+        </a>
+      </div>
+      <div className='pd-wrapper flex jc-center flex-gap flex-wrap'>
+        {products &&
+          products
+            .slice(0, 4)
+            .map((product) => (
```

Comment:
Suggestion: You could use featured as property in the product object to filter in the featured products to show on the homepage.

```
products = [
    { featured: true }
]

products.filter(item => item.featured)
```